### ISSUE TITLE
Accessibility: A11y_Live Chat Widget _Latest Bot Chat Window_ Screenreader: After navigating to minimize button screen reader reads unnecessary “Send button”.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3057]
Copied from #1750235.

[1750235_A11y_Live Chat Widget _Latest Bot Chat Window_ Screenreader.zip](https://github.com/microsoft/BotFramework-WebChat/files/4397049/1750235_A11y_Live.Chat.Widget._Latest.Bot.Chat.Window_.Screenreader.zip)

“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

 

Environment Details:

Application Name: Live Chat Widget

#URL:https://xxx.blob.core.windows.net/portals/democ2adaptivecard.html

Browser Versions: Microsoft Edge Version 80.0.361.69 (Official build) (64-bit)

Windows Version: Windows10

Chrome Version: Version 80.0.3987.149 (Official Build) (64-bit)

 

 

Repro Steps: ​​

1)Launch the application url : https://xxx.blob.core.windows.net/portals/democ2adaptivecard.html
2)Latest Bot icon Button get open and Click on it.
3)"Latest Bot Chat Window" screen get open.
4)Navigate to minimize button.
5)Start NVDA.

Actual:

 

After navigating to minimize button screen reader reads unnecessary “Send button”.

Same issue is not observed with edge+narrator.

Same issue is observed with NVDA for all the Chat Window Screens.

Expected:

It should not announce unnecessary “Send Button” while navigating to minimize button.

User Impact:

Screenreader users will get confused while navigating to Minimize button.

Recommendations: ​

Refer below link which is repository of bug fixes code snippets:

https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

 

MAS Reference:

MAS1.3.1:

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by compulim** [comment]
Maybe related to LCW, need to confirm with Charlie.

**Comment by compulim** [comment]
Close due to no repro.
